### PR TITLE
Adds patting

### PR DESCRIPTION
**Pull request by Thunder12345**
:cl: Thunder12345
add: You can now hug someone while aiming at the head, eyes or mouth to give them a friendly pat
/:cl:

Hugging a person while aiming at the head, eyes or mouth will not give them a pat instead

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
GOOF WAS RIGHT, GOOF WAS ALWAYS RIGHT

**Comment by Aloraydrel** [comment]
Slippy soap

**Comment by Thunder12345** [comment]
The only slippery slope is when you give the clown lube

**Comment by JJRcop** [comment]
Don't you mean "Adds patting"?

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -225,8 +229,6 @@ mob/living/carbon/bullet_act(obj/item/projectile/P, def_zone)
 			resting = 0
 			update_canmove()
 
-		playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
```
Comment (GunHog) [inline code/modules/mob/living/carbon/carbon_defense.dm]: Why delete this? Just move it to the hugging section.

**Comment by Thunder12345** [comment]
I uhh... what...

How did that delete itself

**Comment by PKPenguin321** [comment]
Shaps petting memes have gone too far

**Review by PKPenguin321 — COMMENTED** [review]
Good commit

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -215,7 +215,11 @@ mob/living/carbon/bullet_act(obj/item/projectile/P, def_zone)
 			M.visible_message("<span class='notice'>[M] shakes [src] trying to get them up!</span>", \
 							"<span class='notice'>You shake [src] trying to get them up!</span>")
 		else
-			M.visible_message("<span class='notice'>[M] hugs [src] to make them feel better!</span>", \
+			if(M.zone_selected in list("head", "eyes", "mouth"))
```
Comment (phil235) [inline code/modules/mob/living/carbon/carbon_defense.dm]: `check_zone(M.zone_selected) == "head"`

**Comment by Durkel** [comment]
Where did it all go so wrong?

**Comment by ChangelingRain** [comment]
Please make sure your PR title is accurate.

**Comment by Thunder12345** [comment]
.pet Joan

**Comment by Niknakflak** [comment]
Shaps petting memes have gone too far

**Comment by Tacolizard** [comment]
>Hugging a person while aiming at the **head**, eyes or mouth will not give them a pat instead

>add: You can now hug someone while aiming at the **head** to give them a friendly pat

**Comment by Shadowlight213** [comment]
Dumb meme.

**Comment by Niknakflak** [comment]
Shaps petting memes have gone too far

**Comment by Iamgoofball** [comment]
# SLIPPERY SLOPE YOU CAN'T PROVE ME WRONG

**Comment by optimumtact** [comment]
Lorebus memes are not appropriate for the game and represent a dangerous step towards actively encouraging a known metagroup.

**Comment by Aloraydrel** [comment]
NAMED THE J*W

**Comment by Thunder12345** [comment]
This isn't even related to Lorebus multiple players demanded petting so I coded petting

**Comment by GunHog** [comment]
@KorPhaeron

**Comment by Thunder12345** [comment]
Also I dunno how oranges can call Lorebus a meta group when he's
A: Probably got no idea what happens there
B: Isn't an admin
C: Not aware like half of the regulars there wouldn't know what the server was if it punched them in the face

**Comment by Thunder12345** [comment]
@optimumtact why do you hate fun so much

**Comment by PKPenguin321** [comment]
>multiple players demanded petting so I coded petting

would those players happen to be part of the metagroup? :^)

**Comment by Shadowlight213** [comment]
I support oranges decision. Lorebus is a metagroup. Calling it otherwise is either a lie or being grossly misinformed.

I can fulfill the conditions of A and B, and C is a lie, as the majority of regulars there are either current or past players.

**Comment by Thunder12345** [comment]
Past players aren't people who play on the server by definition. Literally nothing happens there except talk about life and pet Joan, that doesn't mean all forms of petting have to be declared part of the Lorebus conspiracy. And no, it was non Lorebus related people in OOC

**Comment by Shadowlight213** [comment]
The irony here, is that lorebus people are those that are pushing for it.
oranges said no already.
Regardless of the actual intention of the pr, the relationship exists.
The majority of the support for this in the pr being by lorebus members doesn't help.

**Comment by TechnoAlchemisto** [comment]
dangerous shadow cabal

**Comment by kevinz000** [comment]
NOOOOOOOOOOOOOOOOOOOOO

**Comment by Iamgoofball** [comment]
remove metagroups

**Comment by optimumtact** [comment]
You have no idea the depths of my knowledge @Thunder12345 

What I know would shock you all

**Comment by Thunder12345** [comment]
Correlation is not causation

**Comment by Shapsy** [comment]
tfw you start an IRC channel to chat with some friends about common interests and you accidentally turn into the illuminati

**Comment by Thunder12345** [comment]
So that's what the loominarty really is, it's just a group of rich and powerful people who gather to spout memes

**Comment by Kraseo** [comment]
when did people get so scared of social interactions they found headpatting people sexual in any way, shape or form

**Comment by Iamgoofball** [comment]
why is the metagroup showing up to defend this PR all of the sudden

**Comment by Thunder12345** [comment]
Goof half the people supporting this have never been in Lorebus, at least work out who goes there before crying conspiracy

**Comment by GunHog** [comment]
Kor has decided not to overrule Oranges, so sadly, Goof wins.

**Comment by KorPhaeron** [comment]
it's oranges

**Comment by Thunder12345** [comment]
But Kor is a known member of this secret cabal, and thus, his judgement acts to support the cabal. Therefore, we must do exactly the opposite of wha he says, and declare goof wrong.

**Comment by Shadowlight213** [comment]
You've discovered goof's secret weapon. Had Kor reopened the PR, his lorebus affiliation would have simply cemented oranges's point and further justified the closing of the PR.

**Comment by KorPhaeron** [comment]
what if the interests of the cabal include not forcing dumb inside jokes into the codebase and making an embarassing public spectacle

**Comment by Thunder12345** [comment]
Why does everyone even assume Lorebus had anything to do with this? People talked about it I OOC, they said it was be fun, I supplied

**Comment by GunHog** [comment]
**ERRO'S LAW.**

**Comment by TechnoAlchemisto** [comment]
Thanks Kor!

**Comment by kevinz000** [comment]
TESTMERGE!!

**Pull request closed without merging**